routines/epochtal.js

```diff
@@ -265,8 +265,6 @@ async function releaseMap (context) {
 
     fs.renameSync(vmf, `${context.file.vmfs}/${context.data.week.map.id}.vmf.xz`);
 
-    context.data.week.map.file = portal2.map[0];
-
```

Comment:
This is probably just me not understanding the code base, but this seems like a somewhat random line to remove?

Reply:
This is a part of commit 8e686881da66b97d0d63a9919172d2f718e232a4, which made the `file` property universal for all map data objects retrieved from the `workshopper` utility. The file path is required for lobbies to determine if the players have downloaded the map. Without this change, we'd need to duplicate a lot of code to achieve the same thing. Point is, this line simply became redundant.

I should probably take a step back at some point to see how much else was obsoleted with this change, but for now, I'll keep changes to the rest of the system minimal.